src/solvers/td3.ml

```diff
@@ -314,6 +325,7 @@ module WP =
               ignore @@ Pretty.printf "test for %a\n" Node.pretty_trace (S.Var.node x);
               solve x Widen;
               if not (op (HM.find rho x) old_rho) then (
+                print_endline "Destabilization required...";
```

Comment:
Should this printout be there?

Reply:
I think for the time being it's better if they aren't tracing, but also usable normally. It makes interpreting and debugging the interactive benchmark logs easier to see what happened and what didn't.
So making them verbose-only would be more reasonable, but since the rest are normally printed right now anyway, I suppose this can stay for now.